raytracing/imagingpath.py

```diff
@@ -102,6 +102,20 @@ def __init__(self, elements=None, label=""):
         self.showPlanesAcrossPointsOfInterest = True
         super(ImagingPath, self).__init__(elements=elements, label=label)
 
+    @property
```

Comment:
@GabGabG Please take a look at this: @property getter and setter allow for `path.objectHeight = 20`.